Remove comparison with `flutter_emoji` package, because no point of the table is valid anymore. (#32)
GatchHQ/dart-emoji: README.md

👉 A light-weight Emoji 📦 for Dart & Flutter with all up-to-date emojis written in pure Dart 😄 . Made from 💯% ☕ with ❤️!

> This is a fork from [flutter_emoji](https://pub.dev/packages/flutter_emoji), which is inspired from the [node-emoji](https://github.com/omnidan/node-emoji) package. The reason for the fork is better maintaining and adding further features, like `EmojiUtil.hasOnlyEmojis()`.

Reviewer: ```suggestion
This is a fork of [flutter_emoji](https://pub.dev/packages/flutter_emoji), which is inspired by the [node-emoji](https://github.com/omnidan/node-emoji) package. The reason for the fork is better maintaining and adding further features, such as `EmojiUtil.hasOnlyEmojis()`.
```

Reviewer: Not sure about the last sentence in general, but I did not want to rewrite it 😄
